packages/design-system/src/Select/Select.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import RCSelect, { Option as RCOption } from "rc-select";
+import clsx from "classnames";
+import "./rc-styles.css";
+import "./styles.css";
+
+import { Icon } from "Icon";
+import { SelectClassName, SelectDropdownClassName } from "./Select.constants";
+import { Tag } from "Tag";
+import { SelectProps } from "./Select.types";
+
+function Select(props: SelectProps) {
+  const {
+    children,
+    className,
+    dropdownClassName,
+    maxTagCount = 2,
+    maxTagPlaceholder,
+    maxTagTextLength = 5,
+    multiSelect,
```

Comment:
Can we follow the decisions we've made for our api internally and then we can parse and call it appropriately here? For example, we've made the decision of all booleans being prefixed by 'is'.